vms/platformvm/block/builder/helpers_test.go

```diff
@@ -250,31 +234,6 @@ func defaultState(
 	return state
 }
 
-func defaultConfig() *config.Config {
-	return &config.Config{
-		Chains:                 chains.TestManager,
-		UptimeLockedCalculator: uptime.NewLockedCalculator(),
-		Validators:             validators.NewManager(),
-		TxFee:                  defaultTxFee,
-		CreateSubnetTxFee:      100 * defaultTxFee,
-		CreateBlockchainTxFee:  100 * defaultTxFee,
-		MinValidatorStake:      5 * units.MilliAvax,
-		MaxValidatorStake:      500 * units.MilliAvax,
-		MinDelegatorStake:      1 * units.MilliAvax,
-		MinStakeDuration:       configtest.MinStakingDuration,
-		MaxStakeDuration:       configtest.MaxStakingDuration,
-		RewardConfig: reward.Config{
-			MaxConsumptionRate: .12 * reward.PercentDenominator,
-			MinConsumptionRate: .10 * reward.PercentDenominator,
-			MintingPeriod:      365 * 24 * time.Hour,
-			SupplyCap:          720 * units.MegaAvax,
-		},
-		ApricotPhase3Time: genesistest.ValidateEndTime,
-		ApricotPhase5Time: genesistest.ValidateEndTime,
-		BanffTime:         time.Time{}, // neglecting fork ordering this for package tests
```

Comment:
I start changing this partially in this PR and finally fix fork times ordering in next PR